scheds/rust/scx_rusty/src/load_balance.rs

```diff
@@ -784,8 +784,8 @@ impl<'a, 'b> LoadBalancer<'a, 'b> {
                 |task|
                 task.dom_mask
                 & (1 << pull_dom_id) == 1
```

Comment:
I've made the adjustment to make it `!= 0`. I think it's the right case here?
If there're more should be considered I agree that this should be revert and I'll re-work on it. Thanks.

Reply:
I think that's the right fix for that line. Generally speaking tho, both errors do point to changes being made without any verification whether the code is working as intended afterwards. Changes made that way, unless clearly trivial, usually do more harm than good. I think I made a similar request in the past but please let your development guided by actual use cases that you can experiment with. I strongly believe that that would help both the learning process and the resulting code.

Reply:
@htejun -
Thanks for the advise, it really helps a lot, I'll try some workloads like kernel compiling later to validate the code change doesn't harm the scheduler nor does it change the schedulers behavior.
Btw maybe you can suggest some more suitable workloads for the load balancer here ? I'll try to test the scheduler on them as well.

Reply:
As for workload, I think it usually works out the best if there's something that's intrinsically interesting to you, whatever that may be. Current areas of interest seem desktop interactivity, gaming, power saving, server workload latency profile and so on. If you're drawn to an area, it might be useful to find specific workloads or scenarios in that area and try to develop intimate understanding of what the system is doing and how scheduling changes affect them and so on.